src/adlist.c

```diff
@@ -77,6 +77,12 @@ void listRelease(list *list) {
     zfree(list);
 }
 
+/* Just like listRelease, but takes in a into a generic free
+ * function that expects (void *). */
```

Comment:
Some "in a into a" typo here.

```suggestion
/* Just like listRelease, but takes the list as a (void *).
 * Useful as generic free callback. */
```